src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/integration/specs/RoleAssignmentSpecification.groovy

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.keyworker.integration.specs
+
+import org.springframework.http.HttpEntity
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import uk.gov.justice.digital.hmpps.keyworker.dto.RoleAssignmentsSpecification
+
+class RoleAssignmentSpecification extends TestSpecification {
+
+    def "A user that has the MAINTAIN_ACCESS_ROLES_ADMIN role can make role assignment changes"() {
+        given: "That I have the MAINTAIN_ACESS_ROLES_ADMIN role"
+
+        HttpHeaders headers = new HttpHeaders()
+        headers.setBearerAuth(adminToken)
```

Comment:
the `adminToken` doesn't have the `MAINTAIN_ACCESS_ROLES_ADMIN` role though :confused:, it has the `ROLE_MAINTAIN_ACCESS_ROLES` role

Reply:
again that role should not allow access - remove ROLE_MAINTAIN_ACCESS_ROLES only the admin one